Justin Mehmed/homework/week4/day5/consoles/views/consoles_show.erb

```diff
@@ -0,0 +1,64 @@
+<head>
+
+
+
+</head>
+<body>
+
+<h2><%=@consoles['name'] %></h2>
+<h3>Company: <%= @consoles["company"] %></h3>
+<h3>Year: <%= @consoles['year'] %></h3>
+
+<% unless @consoles["image"].nil? || @consoles["image"].empty? %>
+<img src="<%= @consoles["image"] %>" alt="<%= @consoles["name"] %>" class="feature">
+<% end %>
+
+<style>
+
+.container {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+
+.topnav {
+    background-color: #333;
+    overflow: hidden;
+  }
+  
+  .topnav a {
+    float: left;
+    color: #f2f2f2;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    font-family: monospace
+  }
+  
+  .topnav a:hover {
+    background-color: #ddd;
+    color: black;
+  }
+  
+  .topnav a.active {
+    background-color: #4CAF50;
+    color: white;
+  }
+
+  .feature { 
+   max-width: 960px;
+  }
+
+
+
+</style>
+
+
+</body>
+
+
+
+<div class="controls">
+<a href="/consoles_index/<%= @consoles["id"] %>/delete">Delete</a>
+<a href="/consoles_index/<%= @consoles["id"] %>/edit">Edit</a>
+</div>
```

Comment:
Everything should be contained inside the body tags. That being said if you inspected this page when it is rendered you'll notice that it will have two body tags because it will drop into the layout.erb file in place of the yield.  tags.